api/src/pcapi/routes/backoffice/autocomplete.py

```diff
@@ -155,6 +156,19 @@ def _get_venues_base_query() -> sa.orm.Query:
     )
 
 
+def prefill_providers_choices(autocomplete_field: fields.PCTomSelectField) -> None:
+    if autocomplete_field.data:
+        providers = (
+            providers_models.Provider.query.filter(
+                providers_models.Provider.id.in_(autocomplete_field.data),
+                providers_models.Provider.isActive == True,
```

Comment:
[micro-détail] On utilise plutôt `is_()` que `==`, cf. [ce commit](77f2235677cd34a0601899ef8b3c22ca70fb5c5c).

```suggestion
                providers_models.Provider.isActive.is_(True),
```

Reply:
voire juste `providers_models.Provider.isActive` qui permettent d'éviter le `type: ignore` parfois demandé

Reply:
Donc, non ?